xpub.go

```diff
@@ -51,6 +52,11 @@ func (xpub *xpubSocket) Type() SocketType {
 	return xpub.sck.Type()
 }
 
+// Addr returns the listen address
```

Comment:
please adapt with whatever updated `zmq4.Socket.Addr` documentation you came up with.